lib/adf_info.py

```diff
@@ -310,22 +328,36 @@ def __init__(self, config_file, debug=False):
             if cam_ts_done[case_idx]:
                 cam_hist_locs[case_idx] = None
 
+                #Grab case time series file location
+                input_ts_loc = Path(input_ts_locs[case_idx])
+
+                #Get years from pre-made timeseries file(s)
+                found_syear, found_eyear = self.get_climo_yrs_from_ts(input_ts_loc, case_name)
+                found_yr_range = np.arange(found_syear,found_eyear,1)
+
                 #History file path isn't needed if user is running ADF directly on time series.
                 #So make sure start and end year are specified:
-                if syear is None or eyear is None:
-                    msg = f"No given start or end year for {case_name}"
+                if syear is None:
+                    msg = f"No given start year for {case_name}, "
+                    msg += f"using first found year: {found_syear}"
                     print(msg)
-
-                    #Grab case time series file location(s)
-                    input_ts_locs = self.get_cam_info("cam_ts_loc", required=True)
-                    input_ts_loc = Path(input_ts_locs[case_idx])
-
-                    #Get years from pre-made timeseries file(s)
-                    syear, eyear = self.get_climo_yrs_from_ts(input_ts_loc, case_name)
-
-                    msg = f"Using first found year: {syear}\n"
-                    msg += f"and last found year: {eyear}"
+                    syear_baseline = found_syear
+                if syear not in found_yr_range:
+                    msg = f"Given start year '{syear}' is not in current dataset "
+                    msg += f"{case_name}, using first found year: {found_syear}\n"
                     print(msg)
+                    syear_baseline = found_syear
```

Comment:
I think this is supposed to be `syear` here:

```suggestion
                    syear = found_syear
```